src/throttler-module-options.interface.ts

```diff
@@ -1,5 +1,8 @@
+import { ExecutionContext } from '@nestjs/common';
 import { ModuleMetadata, Type } from '@nestjs/common/interfaces';
 
+export type SkipMethod = (context: ExecutionContext, ...param: any) => boolean | Promise<boolean>;
```

Comment:
I guess we can improve the `...param: any` part somehow

Reply:
@micalevisk at first, I defined the Skip method like this:
```ts
export type SkipMethod = (
  context: ExecutionContext,
  req: Record<string, any>,
  res: Record<string, any>,
) => boolean | Promise<boolean>;
```
but the problem is when someone want to override the `handleRequest` method (for example Working with Websockets), the param now is not request and response.
So, I define the method params as `...param: any` so others can use their custom params.